it/neo4j/src/main/java/org/apache/beam/it/neo4j/Neo4jResourceManager.java

```diff
@@ -189,11 +194,26 @@ public String getAdminPassword() {
     return adminPassword;
   }
 
+  private static String formatWaitOption(DatabaseWaitOption waitOption) {
+    if (waitOption instanceof DatabaseWaitInSeconds) {
+      return String.format("WAIT %d SECONDS", ((DatabaseWaitInSeconds) waitOption).getSeconds());
+    }
+    if (waitOption == DatabaseWait.WAIT) {
+      return "WAIT";
+    }
+    if (waitOption == DatabaseNoWait.NO_WAIT || waitOption == null) {
+      return "NOWAIT";
```

Comment:
note: I don't think `NOWAIT` is a good default, but I did this in order to preserve behavior with the current implementation. I wonder if changing the default behavior would be OK here?